sympy/stats/joint_rv_types.py

```diff
@@ -717,3 +719,73 @@ def NegativeMultinomial(syms, k0, *p):
     if not isinstance(p[0], list):
         p = (list(p), )
     return multivariate_rv(NegativeMultinomialDistribution, syms, k0, p[0])
+
+
+# -------------------------------------------------------------------------------
+# Wishart distribution ---------------------------------------------------------
+
+class WishartDistribution(JointDistribution):
+    _argnames = ['n', 'V']
+
+    is_Continuous = True
+
+    @property
+    def set(self):
+        raise NotImplementedError()
```

Comment:
The reason in `NotImplementedError` will be helpful for the user.

Reply:
This is not exactly `NotImplementedError`. I was not getting a way to write this, so thought that I will discuss here.
![support_wishart](https://user-images.githubusercontent.com/34616054/61288768-742d8380-a7e5-11e9-9091-9a7bdcac1e2f.png)
What do you think should I implement this?

Reply:
I think, you need a set of matrices. Wishart distribution is a distribution over matrices rather than real numbers, if I am not wrong. So, find or develop support for sets of random matrices in sympy.